v1.0/create-security-certificates.md

```diff
@@ -113,12 +116,31 @@ If you need to troubleshoot this command's behavior, you can change its [logging
     ~~~
 
     ~~~
-    -rw-r--r--  1 maxroach  maxroach  1.1K Apr 19 09:35 ca.crt
+    total 8
+    -rw-r--r--  1 maxroach  maxroach  1.1K Jul 10 14:12 ca.crt
     ~~~
 
+### Create the certificate and key pair for a client
```

Comment:
This was fine as the last step, keeping node and client certs separate makes things a bit clearer.